mirroroperator/registrymirror.py

```diff
@@ -375,7 +376,15 @@ def generate_stateful_set(self):
         mv $TEMPFILE {shared_cert_mount_path}/{cert_file}
         '''.format(upstream_cert_dir=UPSTREAM_CERT_DIR, cert_file=CERT_FILE,
                    shared_cert_mount_path=SHARED_CERT_MOUNT_PATH)
-
+        script_munge_nameservers = '''
+        cp /var/lib/conf.d/default.conf /var/lib/nameserversdirtemp/default.conf
+        NAMESERVERS=`cat /etc/resolv.conf | grep "nameserver" | awk '{{print $2}}' | tr '\n' ' '`
```

Comment:
can we use `$()` rather than backticks please

Reply:
Sounds good....will update.....thanks